app/components/latest-releases-card.ts

```diff
@@ -0,0 +1,54 @@
+import Component from '@glimmer/component';
+import type CourseExtensionIdeaModel from 'codecrafters-frontend/models/course-extension-idea';
+import type CourseIdeaModel from 'codecrafters-frontend/models/course-idea';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    courseExtensionIdeas: CourseExtensionIdeaModel[];
+    courseIdeas: CourseIdeaModel[];
+  };
+}
+
+interface Release {
+  announcementUrl?: string;
+  releaseDate: Date;
+  title: string;
+  type: 'extension' | 'challenge';
+}
+
+export default class LatestReleasesCardComponent extends Component<Signature> {
+  get releases(): Release[] {
+    const releases: Release[] = [];
+
+    this.args.courseExtensionIdeas.forEach((idea: CourseExtensionIdeaModel) => {
+      if (idea.releasedAt) {
+        releases.push({
+          type: 'extension',
+          title: `${idea.course.shortName} / ${idea.name}`,
+          releaseDate: idea.releasedAt,
+          announcementUrl: idea.announcementUrl || undefined,
+        });
+      }
+    });
+
+    this.args.courseIdeas.forEach((idea: CourseIdeaModel) => {
+      if (idea.releasedAt) {
+        releases.push({
+          type: 'challenge',
+          title: idea.name,
+          releaseDate: idea.releasedAt,
```

Comment:
```suggestion
          releasedAt: idea.releasedAt,
```

Consistent with how we name other date fields